angular.js

```diff
@@ -8,6 +8,17 @@ module.exports = {
     "./rules/karma",
     "./rules/ngrx",
   ].map(require.resolve),
+  plugins: ["@typescript-eslint"],
```

Comment:
Since this extends rules/typescript, its gets typescript-eslint from there, so we don't need to add it here. I don't think it hurts either tho.